pkg/tfgen/generate_schema.go

```diff
@@ -1018,8 +1019,18 @@ func modulePlacementForType(pkg tokens.Package, path paths.TypePath) tokens.Modu
 	}
 }
 
-func parentModule(m tokens.Module) tokens.Module {
-	return tokens.NewModuleToken(m.Package(), parentModuleName(m.Name()))
+func parentModuleOrDefault(m tokens.Module, def tokens.Module) tokens.Module {
```

Comment:
This func only seems to be called with two identical args? And the `m` arg is always overwritten? Can this be cleaned up?

Reply:
Ah sure. Yes happy to clean this up.

Reply:
Done